### PR TITLE
Adds Four Posts Badge!

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request adds one additional badge for when a user takes a fourth action, since there should be 6 badges "visible" to users and we reference 6 badges in our communications about it.

### How should this be reviewed?

👀 

### Any background context you want to provide?

This was meant to be added in to replace the voter registration badge but we forgot to prioritize!!

### Relevant tickets

References [Pivotal #177399018](https://www.pivotaltracker.com/story/show/177399018).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [x] Added appropriate feature/unit tests.
- [x] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [x] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

File: app/Models/Post.php [hunk 1]
```diff
@@ -787,7 +787,13 @@ public function calculatePostBadges()
...
+            if ($userPostsCount >= 4) {
+                $user->addBadge(BadgeType::get('ONE_POST'));
+                $user->addBadge(BadgeType::get('TWO_POSTS'));
+                $user->addBadge(BadgeType::get('THREE_POSTS'));
+                $user->addBadge(BadgeType::get('FOUR_POSTS'));
+            }
+            if ($userPostsCount === 3) {
                 $user->addBadge(BadgeType::get('ONE_POST'));
                 $user->addBadge(BadgeType::get('TWO_POSTS'));
                 $user->addBadge(BadgeType::get('THREE_POSTS'));
```
Comment (DFurnes) [inline app/Models/Post.php:799]: What do you think about changing these to `>=` so we can de-duplicate adding the 1-3 post badges?

```php
if ($userPostsCount >= 4) {
    $user->addBadge(BadgeType::get('FOUR_POSTS'));
}

if ($userPostsCount >= 3) {
    $user->addBadge(BadgeType::get('THREE_POSTS'));
}

if ($userPostsCount >= 2) {
    $user->addBadge(BadgeType::get('TWO_POSTS'));
}

if ($userPostsCount >= 1) {
    $user->addBadge(BadgeType::get('ONE_POST'));
}
```

**Review by DFurnes — APPROVED** [review]
🍀

File: app/Models/Post.php (hunk 1 above)
Comment (lindsayrmaher) [reply app/Models/Post.php:799]: great idea ➕ I should be able to do for tag badges as well.

**Review by DFurnes — APPROVED** [review]
🎉

**Pull request merged**